api/v1beta2/foundationdbcluster_types.go

```diff
@@ -2401,3 +2496,15 @@ func (cluster *FoundationDBCluster) Validate() error {
 
 	return fmt.Errorf(strings.Join(validations, ", "))
 }
+
+// IsTaintFeatureDisabled return true if operator is configured to not replace Pods tainted Nodes OR
+// if operator's TaintReplacementOptions is not set.
+func (cluster *FoundationDBCluster) IsTaintFeatureDisabled() bool {
+	for _, taintOption := range cluster.Spec.AutomationOptions.Replacements.TaintReplacementOptions {
+		// disable taint feature when * taint key has negative or unset DurationInSeconds
+		if pointer.StringDeref(taintOption.Key, "") == "*" && pointer.Int64Deref(taintOption.DurationInSeconds, -1) < 0 {
+			return true
+		}
+	}
+	return len(cluster.Spec.AutomationOptions.Replacements.TaintReplacementOptions) == 0
+}
```

Comment:
```suggestion
func (cluster *FoundationDBCluster) IsTaintFeatureDisabled() bool {
	return len(cluster.Spec.AutomationOptions.Replacements.TaintReplacementOptions) == 0
}
```

From our last discussion we agreed on removing this functionality, any reason to keep it?

Reply:
ah, I didn't recall that. 

I'm fine with this: 
```

func (cluster *FoundationDBCluster) IsTaintFeatureDisabled() bool {
	return len(cluster.Spec.AutomationOptions.Replacements.TaintReplacementOptions) == 0
}
```